tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -140,6 +132,16 @@ def try_handshake(endpoint, port, cipher, ssl_version, server_cert=None, server_
         ocsp=None, sig_algs=None, curves=None, resume=False, no_ticket=False, prefer_low_latency=False, enter_fips_mode=False,
         client_auth=None, client_cert=DEFAULT_CLIENT_CERT_PATH, client_key=DEFAULT_CLIENT_KEY_PATH,
         expected_cipher=None):
+    corked_io_options = [True, False]
+
+    return do_handshake(endpoint, port, cipher, ssl_version, server_cert, server_key, server_cert_key_list, server_cipher_pref,
+            ocsp, sig_algs, curves, resume, no_ticket, prefer_low_latency, enter_fips_mode,
+            client_auth, client_cert, client_key, expected_cipher, corked_io_options[random.randint(0, 1)])
```

Comment:
This is a quick way to get corking coverage, but I would prefer to make it deterministic and not randomly enable/disable it.

Reply:
That was my original implementation in 443d0c4, but it hits 50 mins travis timeout. I have an idea, I could introduce a shell variable use_corked_io=true/false, and config an additional travis build to test corked io. I will update the pr